src/esp/bindings_js/navigate.js

```diff
@@ -12,26 +12,30 @@ class NavigateTask {
   constructor(sim, components) {
     this.sim = sim;
     this.components = components;
-    this.imageCtx = components.canvas.getContext("2d");
-    let shape = this.sim.getObservationSpace("rgb").shape;
+    this.imageCtx = components.canvas.getContext('2d');
+    let shape = this.sim.getObservationSpace('rgb').shape;
     this.imageData = this.imageCtx.createImageData(shape.get(1), shape.get(0));
-    this.semanticCtx = components.semantic.getContext("2d");
-    shape = this.sim.getObservationSpace("semantic").shape;
-    this.semanticImageData = this.semanticCtx.createImageData(shape.get(1), shape.get(0));
+    this.semanticCtx = components.semantic.getContext('2d');
+    shape = this.sim.getObservationSpace('semantic').shape;
+    this.semanticImageData =
+        this.semanticCtx.createImageData(shape.get(1), shape.get(0));
     this.semanticObjects = this.sim.sim.getSemanticScene().objects;
-    components.canvas.onmousedown = e => { this.handleMouseDown(e); }
-    this.radarCtx = components.radar.getContext("2d");
+    components.canvas.onmousedown = e => {
+      this.handleMouseDown(e);
+    } this.radarCtx = components.radar.getContext('2d');
     this.actions = [
-      { name: 'moveForward', key: 'w' },
-      { name: 'lookLeft', key: 'a', },
-      { name: 'lookRight', key: 'd' },
-      { name: 'done', key: ' ' }
+      {name: 'moveForward', key: 'w'}, {
```

Comment:
This formatting doesn't look right. Maybe worth to take a look if there is a flag that needs to be enabled.

Reply:
I was able to fix this a little bit.